app.ts

```diff
@@ -801,11 +807,39 @@ app.get(
 	apisLimiter,
 	isAuthenticated,
 	async function (req, res) {
+		const sensorTypes = configManager.sensorTypes
+		const scales: SensorTypeScale[] = [
+			{
+				type: 'temperature',
+				label: 'Temperature',
+				unit: '°C',
+				group: true,
+			},
+			{
+				type: 'temperature',
+				label: 'Temperature',
+				unit: '°F',
+				group: true,
+			},
+		]
+
+		for (const [, sensor] of sensorTypes) {
+			for (const [, scale] of sensor.scales) {
+				scales.push({
+					type: sensor.key,
+					label: sensor.label,
+					unit: scale.unit,
+					description: scale.description,
+				})
+			}
+		}
```

Comment:
I would unify how those are handled, so you can also use `configManager.namedScales` instead of hardcoding the named ones. Proposal:
- key = scale name or sensor type (key)
- label = scale label
- unit = scale unit
- description: seems unnecessary, unless you display it.